### PR TITLE
Mirror baremetal-machine-controller images

### PR DESCRIPTION
**Pull request by vrutkovs**
PTAL @russellb

### CODE REVIEW

File: cluster/ci/config/mirroring/origin_v4_0_quay [hunk 1]
```diff
@@ -1,6 +1,7 @@
 registry.svc.ci.openshift.org/openshift/origin-v4.0:ansible quay.io/openshift/origin-ansible:v4.0 quay.io/openshift/origin-ansible:v4.0.0 quay.io/openshift/origin-ansible:latest
 registry.svc.ci.openshift.org/openshift/origin-v4.0:artifacts quay.io/openshift/origin-artifacts:v4.0 quay.io/openshift/origin-artifacts:v4.0.0 quay.io/openshift/origin-artifacts:latest
 registry.svc.ci.openshift.org/openshift/origin-v4.0:aws-machine-controllers quay.io/openshift/origin-aws-machine-controllers:v4.0 quay.io/openshift/origin-aws-machine-controllers:v4.0.0 quay.io/openshift/origin-aws-machine-controllers:latest
+registry.svc.ci.openshift.org/openshift/origin-v4.0:baremetal-machine-controllers docker.io/openshift/origin-baremetal-machine-controllers:v4.0 docker.io/openshift/origin-baremetal-machine-controllers:v4.0.0 docker.io/openshift/origin-baremetal-machine-controllers:latest
```
Comment (russellb) [inline cluster/ci/config/mirroring/origin_v4_0_quay]: Looks like docker needs to change to quay in this one

Comment (vrutkovs) [reply cluster/ci/config/mirroring/origin_v4_0_quay]: Good catch, fixed

**Comment by smarterclayton** [comment]
I sent an email out a while back.  You also need to have ART specific repos and get a public repo from comet.

New operators may not be added to the payload at this point unless they follow this process.

**Pull request merged**